UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling/ChangelingAbility/AbsorbAbility.cs

```diff
@@ -0,0 +1,116 @@
+using HealthV2;
+using Items.Implants.Organs;
+using System;
+using System.Collections;
+using UnityEngine;
+
+namespace Changeling
+{
+	[CreateAssetMenu(menuName = "ScriptableObjects/Systems/ChangelingAbilities/AbsorbAbility")]
+	public class AbsorbAbility : Sting
```

Comment:
why inherit from ``Sting``?

Reply:
Because absorb have some same parts as GetPlayerOnClick method and they actually doing same thing